crates/cheatcodes/src/evm.rs

```diff
@@ -298,7 +301,23 @@ impl Cheatcode for getBlockTimestampCall {
         Ok(ccx.data.env.block.timestamp.abi_encode())
     }
 }
+impl Cheatcode for blobBaseFeeCall {
+    fn apply_full<DB: DatabaseExt>(&self, ccx: &mut CheatsCtxt<DB>) -> Result {
+        let Self { newBlobBaseFee } = self;
+
+        ensure!(
+            ccx.data.env.cfg.spec_id >= SpecId::CANCUN,
+            "`blobBaseFee` is not supported before the Cancun hard fork; \
+             see relevant EIP for blob base fee"
+        );
 
+        let excess_blob_gas = (*newBlobBaseFee).to_ethers();
+        let c: u64 = excess_blob_gas.to_string().parse().unwrap();
```

Comment:
this is not necessary and can be replaced with a `.to()` call

Reply:
> this is not necessary and can be replaced with a `.to()` call

oh yeah correct thanks for this